custom-tests.json

```diff
@@ -41,6 +41,9 @@
     },
     "EXT_texture_filter_anisotropic": {
       "__base": "var canvas = document.createElement('canvas'); if (!canvas) {return false}; var gl = canvas.getContext('webgl'); if (!gl) {return false}; var instance = gl.getExtension('EXT_texture_filter_anisotropic');"
+    },
+    "MediaDevices": {
+      "__base": "var instance = navigator.mediaDevices;"
```

Comment:
FWIW, this is something we should be able to derive from just the IDL, since it's just accessing attributes starting at the global object. But  no harm in testing it manually until then!